package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webdesign-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
Why "dependencises" modules are needed under root?

Reply:
Didn't even realize it was there,

removed it